src/org/usfirst/frc/team2706/robot/subsystems/TrackerBox2.java

```diff
@@ -1,42 +1,43 @@
 package org.usfirst.frc.team2706.robot.subsystems;
```

Comment:
In the future, try not to mess with TrackerBox2, because it is just a file copied over from a different repository.